### PR TITLE
docs: update roadmap as of q4 2022

### PR DESCRIPTION
**Pull request by mgechev**

### CODE REVIEW

File: aio/content/guide/roadmap.md [hunk 1]
```diff
@@ -14,46 +14,34 @@ For example, features are released in the next minor after they are complete, or
...
-The feature lets developers augment their own components with additional behaviors without using inheritance.
-The project requires substantial effort in terms of the definition of APIs, semantics, and implementation.
+As part of this initiative we are going to implement automatic import of template dependencies for standalone components. Additionally, to enable smaller application bundles the language service will propose automatic removal of unused imports.
 
-### Better stack traces
+### Investigate modern bundles
 
-The Angular and the Chrome DevTools are working together to enable more readable stack traces for error messages.
+To improve development experience by speeding up build times research modern bundles.
+As part of the project experiment with [esbuild](https://esbuild.github.io) and other open source solutions, compare them with the state-of-the-art tooling in Angular CLI, and report the findings. In Angular v15 we have experimental esbuild support in `ng build` and `ng serve`. We'll continue iterating on the solution until we're confident to release it as stable.
```
Comment (alan-agius4) [inline aio/content/guide/roadmap.md:40]: We don’t yet have esbuild support in `ng serve`.

File: aio/content/guide/roadmap.md [hunk 2]
```diff
@@ -14,46 +14,34 @@ For example, features are released in the next minor after they are complete, or
...
-A [long-standing feature request](https://github.com/angular/angular/issues/8785) is to add the ability to add directives to host elements.
-The feature lets developers augment their own components with additional behaviors without using inheritance.
-The project requires substantial effort in terms of the definition of APIs, semantics, and implementation.
+As part of this initiative we are going to implement automatic import of template dependencies for standalone components. Additionally, to enable smaller application bundles the language service will propose automatic removal of unused imports.
 
-### Better stack traces
+### Investigate modern bundles
 
-The Angular and the Chrome DevTools are working together to enable more readable stack traces for error messages.
+To improve development experience by speeding up build times research modern bundles.
```
Comment (jelbourn) [inline aio/content/guide/roadmap.md:39]:
```suggestion
To improve development experience by speeding up build times, we plan to explore options to improve JavaScript bundles created by Angular CLI.
```

File: aio/content/guide/roadmap.md [hunk 3]
```diff
@@ -63,23 +51,33 @@ We are evaluating components in Angular Material against accessibility standards
...
+### Improvements in the image directive
+
+We released the Angular [image directive](https://developer.chrome.com/blog/angular-image-directive/) as stable in v15. As the next step, we'll be working on fill mode that will enable images to fit within their parent container rather than having explicit dimensions. We'll be also working on automatic `srcset` generation so that users can download the smallest image based on the required dimensions.
+
 ## Future
 
-### Explore hydration and server-side rendering usability improvements
+### Token-based theming APIs
+
+To provide better customization of our Angular material components and enable Material 3 capabilities, we'll be collaborating with the Google Material team on defining token-based theming APIs.
```
Comment (jelbourn) [inline aio/content/guide/roadmap.md:70]:
```suggestion
To provide better customization of our Angular material components and enable Material 3 capabilities, we'll be collaborating with Google's Material Design team on defining token-based theming APIs.
```

**Review by jelbourn — APPROVED** [review]
LGTM

File: aio/content/guide/roadmap.md [hunk 4]
```diff
@@ -63,23 +51,33 @@ We are evaluating components in Angular Material against accessibility standards
...
+For the past couple of quarters we understood and defined the problem space. We are going to follow up with a series of blog posts on best practices when developing applications at scale.
+
+### Update getting started tutorial
+
+We're working on updating the Angular getting started experience with standalone components. As part of this initiative, we'd like to create a new textual and video tutorials.
+
+### Improvements in the image directive
+
+We released the Angular [image directive](https://developer.chrome.com/blog/angular-image-directive/) as stable in v15. As the next step, we'll be working on fill mode that will enable images to fit within their parent container rather than having explicit dimensions. We'll be also working on automatic `srcset` generation so that users can download the smallest image based on the required dimensions.
+
```
Comment (AndrewKushnir) [inline aio/content/guide/roadmap.md:65]: Both of these features (the `fill` mode and an automatic `srcset` generation) are implemented and will be a part of v15 (the `fill` mode feature would retain the developer preview status for now, to collect more feedback before making it stable).

File: aio/content/guide/roadmap.md [hunk 5]
```diff
@@ -14,46 +14,34 @@ For example, features are released in the next minor after they are complete, or
...
 
-### Investigate modern bundles
+### Improve documentation and schematics for standalone components
 
-To improve development experience by speeding up build times research modern bundles.
-As part of the project experiment with [esbuild](https://esbuild.github.io) and other open source solutions, compare them with the state-of-the-art tooling in Angular CLI, and report the findings. In Angular v14 we're releasing an [experimental support](https://github.com/angular/angular-cli/pull/22995) for esbuild. Next, the team will focus on validating the new prototype and implementing watch and Sass support.
+We are working on developing schematics for applications bootstrapped with a standalone component. Additionally, we are filling the documentation gaps of the simplified standalone component APIs.
```
Comment (AndrewKushnir) [inline aio/content/guide/roadmap.md:27]: We can probably mention that we plan to work on extending the `ng new` to add an ability to create an app using standalone bootstrap APIs (in addition to the current mode of creating an app that uses NgModule-based APIs).

Comment (mgechev) [reply aio/content/guide/roadmap.md:27]: That's what I meant by schematics for applications bootstrapped with a standalone component. I'll change to `ng new` collection to keep it clearer.

File: aio/content/guide/roadmap.md [hunk 6]
```diff
@@ -114,6 +108,43 @@ As part of this project we'd like to implement mixed orientation support for the
   </summary>
   <div class="details-content">
 
+### Improve image performance
+
+*Completed Q4 2022*
+
+The [Aurora](https://web.dev/introducing-aurora/) and the Angular teams are working on the implementation of an image directive that aims to improve [Core Web Vitals](https://web.dev/vitals). We shipped a stable version of the image directive in v15.
```
Comment (AndrewKushnir) [inline aio/content/guide/roadmap.md:115]:
```suggestion
The [Aurora](https://web.dev/introducing-aurora/) and the Angular teams are working on the implementation of an image directive that aims to improve [Core Web Vitals](https://web.dev/vitals). We shipped a stable version of the image directive in v15. More information about the Image Directive can be found in [this guide](https://angular.io/guide/image-directive).
```

**Review by AndrewKushnir — APPROVED** [review]
LGTM, just a quick proposal to add a link to the image directive guide.

**Comment by alxhub** [comment]
This PR was merged into the repository by commit 0bc44058602b8844ba69aa7b34d81839954a1d90.

**Pull request closed without merging**